### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by litozpl** [fat-tire/resolve#38] [Q&A]
Hello, It's me again! :)

My new issue is about exportation.
When I want to deliver, I got a message "Please select a render path from within the media storage.". I've change various times the path, inside "resolve-main" and in an mount folder. But nothing's going well.
When I do Proxy or Optimized media, everything looks ok.
I tried "quick export", It doesn't work. I got a endless waiting box, rendering at 0fps.

Do you have that kind of issue ?

Thanks!

**Comment by fat-tire** [comment]
I haven't seen this issue.  Are you using 18.5.1?

Do you possibly have a permission error?  That is, does the container have the ability to write a file in whatever folder you are selecting?  Also, if you change codecs, does this affect the outcome?

**Reply by litozpl** [reply]
I have 18.5b5, not 18.5.1. May be an beta issue ? I will download the new version.

I don't no if there is a permission error. DaVinci can save and export the project, make proxy and consolidate files in whatever folder. So, I don't think.
Same if I change codecs.

**Comment by fat-tire** [comment]
Hm.  Are you able to write files from the shell?  That is, from the host do:

`./resolve.sh /bin/bash`

and then from the shell inside the container, use the command

`touch /path/to/where/its/not/saving/newfile.txt`

does this give you an error?  (if it works, it will create a new file called `newfile.txt` at the path with a size of zero.  You can delete it after the test.)

The error may give you information about what is wrong..  Are you using podman or docker?

**Reply by litozpl** [reply]
Ok, so it doesn't have permission to write.

I use Podman. I have to give right to Podman ?

**Reply by litozpl** [reply]
It's ok!
I've change all path everywhere in all menu's possible, and It's working :)
I don't understand why, but you know...
Thanks again for taking time to help me!

**Comment by fat-tire** [comment]
hahah glad you figured it out!

(I use podman too!)

I'm glad it's working now--  maybe it was a permissions issue or ownerships on the folder itself?  :shrug:
